python/dlr/tf_model.py

```diff
@@ -50,17 +51,20 @@ def _get_input_and_output_names(graph):
     output_tensor_names = set()
     op_prefix = PREFIX + "/"
     for op in graph.get_operations():
-        # Ignore operators which are not directly under PREFIX node
-        if not op.name.startswith(op_prefix) or op.name.find('/', len(op_prefix)) > -1:
```

Comment:
you removed `op.name.find('/` so the `UNLIKELY_OUTPUT_TYPES` and `control_inputs` help fix the multi-level issue you met before?

Reply:
New code correctly found output tensor on 3rd level for mobilenet model
```
output_tensor_names: ['import/MobilenetV1/Predictions/Reshape_1:0']
```
Also new code filtered out tensors used as control inputs in ssd_mobilenet model
```
# good outputs
'import/detection_boxes:0'
'import/detection_classes:0'
'import/detection_scores:0'
'import/num_detections:0'

#filtered out outputs (they are "control_inputs" for other ops/tensors)
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond_3/switch_t:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond_3/switch_f:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond_3/cond/switch_t:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond_1/switch_t:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond/switch_t:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond_1/cond/switch_f:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond/cond/switch_t:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond/switch_f:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond_3/cond/switch_f:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond_1/cond/switch_t:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond_1/switch_f:0'
'import/Postprocessor/BatchMultiClassNonMaxSuppression/map/while/PadOrClipBoxList/cond/cond/switch_f:0'
```